inflate_fast_amd64_test.go

```
@@ -0,0 +1,800 @@
+			var w deflateTestBits
+			w.write(1, 1)
+			w.write(1, 2)
+			tt.write(&w)
+			src := append(w.bytes(), make([]byte, 40)...)
+
+			d, r := amd64PrepareHuffmanBlock(t, src, 1)
+			const dstLen = deflateFastOutputMargin + 1
+			reference := amd64RunHuffmanGo(d, r, dstLen)
+			kernel, statuses := amd64RunHuffmanKernel(t, d, r, dstLen)
+
+			if reference.err != errDeflateBadData || kernel.err != reference.err {
```

golangci-lint (2.12.2)

[error] comparing with != will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)